index.js

```diff
@@ -51,10 +51,11 @@ StateMachine.prototype.trigger = function (name, cb) {
 
   var target = transitions.find(function (transition) {
     if (
-        (Array.isArray(transition.from) && transition.from.indexOf(self.state) !== -1)
-        || transition.from === self.state){
+      (Array.isArray(transition.from) && transition.from.indexOf(self.state) !== -1) ||
+        transition.from === self.state ||
+        transition.from === WILDCARD) {
```

Comment:
The logic is wrong. Now if the wildcard is encountered before an exact match, it will be executed. The wildcard should be a fallback.

See the line below: https://github.com/kapetan/state-transition/blob/67025385e4618b15f8a160074adc00f8c7d90e43/index.js#L55

Reply:
Already spotted... and fixed